tests/integration/product-type-export-encoding.spec.js

```diff
@@ -72,9 +72,14 @@ test(`productType export module
   const expectedResult2 = 'name,key,description,breite\n'
     + 'custom-product-type,productTypeKey,Some description - žluťoučký kůň úpěl ďábelské ódy,X\n'
 
-  const expectedEncoded1 =
+  let expectedEncoded1 =
     'name,type,attributeConstraint,isRequired,isSearchable,label.en,label.de,textInputHint,displayGroup\n'
     + 'breite,number,None,false,false,EN:�lu�ou�k� k�� �p�l ��belsk� �dy,DE:�=�������=����������,SingleLine,Other\n'
+
```

Comment:
Could we please add a `todo` comment to fix this when we change to node versions 8 and 10? I think this is just a temporary workaround since the differences in encoding between node 6 and 8, right?

```js
// TODO use only one expect string when Node 10 LTS support kicks in
```

More reading on the release schedule here:
https://github.com/nodejs/Release#release-schedule